examples/demo.yaml

```diff
@@ -78,7 +78,7 @@ spec:
     spec:
       containers:
       - name: curl
-        image: appropriate/curl
```

Comment:
The `appropriate/curl` image uses `alpine:3.7` which is quite old. There are known vulnerabilities in that image (see [here](https://hub.docker.com/r/alpine/curl)).